code/modules/clothing/neck/_neck.dm

```diff
@@ -291,3 +291,110 @@
 	desc = "A sizable white cape with gold connects."
 	icon_state = "grandadmiral"
 	item_state = "grand_admiral"
+
+/*
+ * Bodycamera stuff
+ * NOTE: Unlike regular cameras, bodycams can only stream to one camera network at a time. Unless you want to give it more networks,
+ * the code will only ever worry about bodcam.network[1]
+ */
+
+/obj/item/clothing/neck/bodycam
```

Comment:
You need to setup somewhere a thing to call updatePortableCamera on movement, see https://github.com/yogstation13/Yogstation/blob/master/code/modules/mob/living/silicon/silicon_movement.dm for an example(altho you need to also hook somewhere to also update when the person wearing it moves since the location when equipped is the person, not the turf the person is standing on)

Reply:
It works as is perfectly in testing tho :(

Reply:
I am willing to implement this if you help me but it works without this already, what benefits would this even have

Reply:
yeah I tested it again, using 2 logins, one watching, one running around. Went on shuttles, jumped in disposals, went into space, ran around, dropped the cam, threw the cam, held a cam in both hands, and it never ever worked in a way not intended. I think updatePortableCamera is not needed here

Reply:
ok so here is what I think:
updatePortableCamera is needed for smooth movement. Normally byond updates client.eye every tick, making the mob movement appear a little stuttery. That would be the benefit as far as I am aware of doing something like this.